.circleci/config.yml

```diff
@@ -60,26 +60,31 @@ docker_import: &docker_import
         -i /docker/$APP_NAME-latest.tar
 
 docker_push: &docker_push
-  run:
-    name: Push commit image to dockerhub
+  run: 
+    name: Push docker images to ECR
     command: |
-      docker login -u $DOCKER_USER -p $DOCKER_PASS
-      docker push \
-        nerveshub/$APP_NAME:$DOCKER_TAG
+      aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws/o0z1n0s8
+      docker push public.ecr.aws/o0z1n0s8/nerveshub/$APP_NAME:$DOCKER_TAG
+
+docker_login: &docker_login
+  aws-cli/setup:
+    aws-access-key-id: AWS_ACCESS_KEY
+    aws-secret-access-key: AWS_ACCESS_SECRET
+    aws-region: AWS_REGION_NAME        
 
 docker_release: &docker_release
   run:
     name: Release docker images to dockerhub
     command: |
-      docker login -u $DOCKER_USER -p $DOCKER_PASS
       docker push \
-        nerveshub/$APP_NAME:$DOCKER_VERSION_TAG
+        public.ecr.aws/o0z1n0s8/nerveshub/$APP_NAME:$DOCKER_VERSION_TAG
       docker push \
-        nerveshub/$APP_NAME:latest
+        public.ecr.aws/o0z1n0s8/nerveshub/$APP_NAME:latest
 
 version: 2.1
 orbs:
   docker: circleci/docker@2.0.3
+  aws-cli: circleci/aws-cli@2.0  
```

Comment:
I think [circleci/aws-cli@3.1.3](https://circleci.com/developer/orbs/orb/circleci/aws-cli) is the latest orb - any reason not to use it?